svsim/src/main/scala/Simulation.scala

```diff
@@ -41,6 +42,22 @@ final class Simulation private[svsim] (
       processBuilder.environment().put(pair._1, pair._2)
     }
     val process = processBuilder.start()
+    // Fire and forget a thread that forwards the process's stderr.
+    // This thread will block on the processes error stream until it is closed, then the thread will complete.
+    implicit val e: ExecutionContext = ExecutionContext.global
+    Future {
+      // TODO use transferTo once we require Java 9 or newer.
+      val BufSize = 8192 // Taken from java.io.InputStream.DEFAULT_BUFFER_SIZE.
+      val err = process.getErrorStream()
+      val buffer = new Array[Byte](BufSize)
+      var read = 0
+      while ({
+        read = err.read(buffer, 0, BufSize)
+        read >= 0
+      }) {
+        stderrStream.write(buffer, 0, read)
+      }
+    }
```

Comment:
kind of a drive-by comment, but is there a particular reason why you're handling this as a Java Stream rather than using Scala's nice streaming data-types and stream programming constructs? Is there non Scala equivalent to `Console.err` that you can use?

Reply:
I think Scala's streaming constructs are generally for streaming data, whereas for process-level I/O streams I think it's still pretty typical to just use `java.io` streams (as evidenced by the fact that scala.Console.err is a `java.io.OutputStream`). I guess I'm not aware of any advantage to wrapping this in a Scala type but if anyone has a suggestion I'm all ears 🙂.

I could use `System.err`, I'm not really sure when one should use Java's `System` vs. Scala's `Console` for these sorts of things, but we often use things like `Console.withOut` to capture stdout, so I figured I'd go with the Scala API.